tools/xc7frames2bit.cc

```diff
@@ -53,7 +57,9 @@ struct Frames2BitWriter {
 		    std::is_same<ArchType, xilinx::UltraScalePlus>::value) {
 			// In case the frames input file is missing some frames
 			// that are in the tilegrid
-			frames.addMissingFrames(part);
+			if (FLAGS_partial_bitstream != "true") {
```

Comment:
This is something that was absent in`xc7patch`. Is it a correct approach to disable this step when we want to generate partial bitstream?

Reply:
It depends. Adding missing (empty) frames is crucial when creating a bitstream with configuration data utilizing the Type2 packets. I guess the partial bitstream doesn't span over thw whole FPGA and only touches a sinlge column or so. In this case we should be fine. But if something goes wrong and doesn't work on hardware this would be something to keep in mind.